_assignments/mp1.md

```diff
@@ -372,9 +375,9 @@ To change the light, write something similar to the following, but remember to u
 ![Get Tab Key Down](/img/assignments/mp1/image14.png)
 
 For the actual light color, you can either create a new color using the [`new Color(red, green, blue)`](https://docs.unity3d.com/ScriptReference/Color-ctor.html) constructor, or one of the [predefined colors](https://docs.unity3d.com/ScriptReference/Color.html).
-How you change the light is up to you, but pressing the controller button you designate must visibly change the light color.
+How you change the light is up to you, but pressing the light switch controller button repeatedly must visibly change the light color each time.

```

Comment:
This feels really syntactically strange.

Reply:
I can rephrase it. All I want to communicate is that repeatedly pressing the button should repeatedly change the color.